app/services/dttp/retrieve_contact.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Dttp
+  class RetrieveContact
```

Comment:
This should be consistently named with the other two that @JonF1  and @avinhurry are doing. I'd probably go with `Dttp::Contact::Fetch`, `Dttp::PlacementAssignment::Fetch`, `Dttp::Account::Fetch` and then each of those returns a `Dttp::Contact`, `Dttp::PlacementAssignment`, `Dttp::Account`.